### ISSUE TITLE
wasm-interp: support running wasm binaries created by `wasm-opt -ttf`?

### ISSUE THREAD

**Issue by fitzgen** [WebAssembly/wabt#1094]
The `-ttf` option to `wasm-opt` stands for "translate to fuzz" and essentially makes wasm-opt into a test case generator for fuzzing.

However, it generates imports that `wasm-interp` doesn't know about, but are morally equivalent to `host.print` AFAICT:

```wasm
 (import "fuzzing-support" "log-i32" (func $log-i32 (param i32)))
 (import "fuzzing-support" "log-i64" (func $log-i64 (param i64)))
 (import "fuzzing-support" "log-f32" (func $log-f32 (param f32)))
 (import "fuzzing-support" "log-f64" (func $log-f64 (param f64)))
```

Because `wasm-interp` doesn't support these imports, one can't run these wasm binaries in `wasm-interp`.

Would supporting these imports in `wasm-interp` be something y'all are interested in?

**Comment by binji** [comment]
Seems like a good idea. Maybe the best solution is to provide a way to log all imports, and have them return dummy values.

**Comment by binji** [comment]
Fixed in #1101.
